bootstrap/image_registries.yaml

```diff
@@ -4,6 +4,6 @@
 registries:
   - name: kubeflow
```

Comment:
Why are you upgrading this file? Can you revert it back to the original value and not use kkassarvi?

Here's the process I follow to do debugging

1. Follow kubeflow.org to deploy on GCP
2. Run this command to copy your ksonnet app to your local machine
https://www.kubeflow.org/docs/started/getting-started-gke/
3. Replace vendor/kubeflow with a symbolic link to your code; this way you will pick up changes to your ksonnet component
4. Recreate the K8s resources to pick up your changes.